packages/web/src/db/adapters/wa-sqlite/WASQLiteDBAdapter.ts

```diff
@@ -103,6 +107,8 @@ export class WASQLiteDBAdapter extends BaseObserver<DBAdapterListener> implement
           : getWorkerDatabaseOpener(this.options.dbFilename, enableMultiTabs, optionsDbWorker);
 
       this.methods = await dbOpener(this.options.dbFilename);
+
+      await this.methods?.execute(tempStoreQuery);
```

Comment:
```suggestion
      await this.methods!.execute(tempStoreQuery);
```

I prefer the `!` version here - makes it clear that it will always be executed.